src/main/java/org/ow2/proactive/catalog/dto/CatalogObjectMetadata.java

```diff
@@ -50,6 +50,9 @@
 @EqualsAndHashCode(callSuper = false)
 public class CatalogObjectMetadata extends ResourceSupport {
 
+    @JsonProperty("commit_time_raw")
+    protected final String commitTimeRaw;
```

Comment:
So it means that we will have commit_time_raw and commit_time. From the beginning when object was posted these times will be the same. But when we restore revision we update commit_time. Right?
Was it the needed requirement to have both dates? Why not to use only commit_time?